### ISSUE TITLE
[Project 3] Question about sched_setscheduler

### ISSUE THREAD

**Issue by Kangji** [swsnu/osspr2021#88]
안녕하세요. sched_setscheduler 관련해서 궁금한 점이 생겨서 질문 남깁니다.
SCHED_NORMAL이나 SCHED_BATCH task도 setweight/getweight에서 -EINVAL을 return해야 하나요?
감사합니다.

**Comment by WoosukKwon** [comment]
Yes. The `sched_setweight` and `sched_getweight` system calls should only work with WRR tasks (i.e., tasks with the `SCHED_WRR` policy). Otherwise, you should return an error.

**Comment by jnooree** [comment]
그런데 원래 패치에서는 `SCHED_NORMAL`을 전부 `SCHED_WRR`로 돌려버렸던 것으로 기억하는데, 그렇다면 `wrr_sched_class`에 해당하는 task는 모두 가능하게 해도 되는 것이 아닌가요..?

**Comment by WoosukKwon** [comment]
Tasks with `SCHED_WRR` policy will be assigned to the WRR scheduler, and the scheduling policy of the tasks managed by the WRR scheduler will be `SCHED_WRR`. Thus, you can check it in either way.

**Comment by jnooree** [comment]
감사합니다!

**Comment by Kangji** [comment]
그러면 Task를 fork할 때 부모 task가 sched_normal이나 sched_batch여도 sched_wrr로 바꿔줘야 하는 건가요?

**Comment by WoosukKwon** [comment]
Actually, because there must be no CFS tasks in the system, we don't care about that.

**Comment by Kangji** [comment]
감사합니다!
